src/app/api/partners/[slug]/audit/route.ts

```diff
@@ -166,11 +166,11 @@ export async function GET(
         if (arweaveArticle?.content?.title) {
           info.push(`Arweave article title: ${arweaveArticle?.content?.title}`)
         } else {
-          error.push(`Arweave article not found`)
+          warn.push(`Arweave article not found`)
```

Comment:
It'd be good if we have an else condition on the first if here (i.e. the equivalent of `if (!partner. aarweaveDigest)` and push an error for Arweave Digest not set